fixtures/ext-types/guid/src/lib.rs

```diff
@@ -24,15 +24,15 @@ fn get_guid_helper(vals: Option<GuidHelper>) -> GuidHelper {
             ],
             maybe_guid: None,
         },
-        Some(vals) => vals,
+        Some(values) => values,
     }
 }
 
 impl UniffiCustomTypeWrapper for Guid {
     type Wrapped = String;
 
-    fn wrap(val: Self::Wrapped) -> uniffi::Result<Self> {
-        Ok(Guid(val))
+    fn wrap(values: Self::Wrapped) -> uniffi::Result<Self> {
```

Comment:
nit, but seeing you changed the name, the singular `value` seems more appropriate here?